src/dropdown/dropdown.scss

```diff
@@ -94,6 +94,11 @@
       color: inherit;
       opacity: 1;
     }
+    
+    .color-text-secondary,
+    .color-text-secondary {
+      color: var(--color-state-hover-primary-text) !important;
+    }
```

Comment:
I wonder if we should make it more general so that any of the text color utilities will be the correct color on hover? Also curious if `inherit` will work here.

```suggestion
    [class*="color-text-"] {
      color: inherit !important;
    }
```

Reply:
Oh cool! Is `[class*=` a thing that SCSS handles, or is it just broadly supported in different browsers with plain CSS?

I'll try this change in my code, see if it fixes the text appearance in the same way. 👍🏻

Reply:
It's a fuzzy attribute selector https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors

Reply:
Yeah, that looks like it works!

![image](https://user-images.githubusercontent.com/82317/124326185-ac9e0780-db4b-11eb-9d97-4896b7311f95.png)